.expeditor/config.yml

```diff
@@ -25,6 +25,9 @@ subscriptions:
   - workload: project_promoted:chef/automate:master:acceptance:*
     actions:
     - bash:.expeditor/update_hugo_modules_project_promoted.sh
+  - workload: project_promoted:habitat:acceptance:*
```

Comment:
This should actually be:
```
- workload: project_promoted:habitat:current:*
```
...as the channel before _stable_ is _current_. See more at:
https://github.com/habitat-sh/habitat/blob/master/.expeditor/config.yml#L136-L148